systray_menu_unix.go

```diff
@@ -224,6 +224,34 @@ func addSeparator(id uint32, parent uint32) {
 	refresh()
 }
 
+func hideSeparator(id uint32, parent uint32) {
+	instance.menuLock.Lock()
+	defer instance.menuLock.Unlock()
+	menu, _ := findLayout(int32(parent))
+	for _, i := range menu.V2 {
+		item := i.Value().(*menuLayout)
+		if item.V0 == int32(id) {
+			item.V1["visible"] = dbus.MakeVariant(false)
+			refresh()
+			return
+		}
+	}
+}
+
+func showSeparator(id uint32, parent uint32) {
+	instance.menuLock.Lock()
+	defer instance.menuLock.Unlock()
+	menu, _ := findLayout(int32(parent))
+	for _, i := range menu.V2 {
+		item := i.Value().(*menuLayout)
+		if item.V0 == int32(id) {
+			item.V1["visible"] = dbus.MakeVariant(true)
+			refresh()
+			return
+		}
+	}
```

Comment:
This function is exactly the same as hideSeparator only with the argument to dbus.MakeVariant() differ. Could the code be reused?